### ISSUE TITLE
Run only once?

### ISSUE THREAD

**Issue by davidnewhall** [wailsapp/wails#2477]
### Have you read the Documentation Contribution Guidelines?

- [X] I have read the [Documentation Contribution Guidelines](https://wails.io/community-guide#ways-of-contributing).

### Description

This isn't really an issue, but more of a question, and possibly a feature request.

If I double-click a wails-built app twice (exe in this case), it will open twice. Does Wails have anything to prevent that from happening? Running more than once may be dangerous in my case. I'm contemplating writing a file with a pid I can check on startup. Got any other ideas or suggestions?

I checked the docs, and searched the GH issues, but I don't see anything about this. If it's a documentation issue, I'm happy to fix it (once a solution is identified).

Thank you!

### Self-service

- [X] I'd be willing to address this documentation request myself.

**Comment by stffabi** [comment]
Thanks for opening this issue and using Wails.

Unfortunately we currently don't have an integrated solution for this. We have an issue for this feature #1351, where also some people mentioned their solutions for it.

#1351 is currently not on the near roadmap (sometime post v3).

**Comment by stffabi** [comment]
Duplicate of #1351

**Comment by davidnewhall** [comment]
Thank you for the links!
